deployments/ecs/agent.yaml

```
@@ -56,6 +57,19 @@ monitors:
     # config for each EC2 instance.
   - {"#from": "env:EXTRA_MONITOR_CONFIG", default: [], flatten: true}
 
+  # Enables the Smart Agent to receive traces.
+  - type: signalfx-forwarder
+    # Specify the URL:port where the Smart Agent listens for traces.
+    listenAddress: "0.0.0.0:9080/v1/trace"
```

```suggestion
    listenAddress: "0.0.0.0:9080"
```

Sorry, I didn't notice this before.